islandora.module

```diff
@@ -397,7 +397,14 @@ function islandora_menu() {
     'type' => MENU_NORMAL_ITEM,
     'access arguments' => array(ISLANDORA_MANAGE_DELETED_OBJECTS),
   );
-
+  $items['admin/reports/orphaned_objects/list'] = array(
+    'title' => 'Orphaned Islandora objects',
+    'description' => 'List of orphaned Islandora objects.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('islandora_orphaned_objects_manage_form'),
+    'access arguments' => array('manage orphaned objects'),
```

Comment:
Did this permission ever get defined in islandora.module?

Reply:
Failed to add that to my commit somehow! fixed.